docs/source/installation/upgrade.rst

```diff
@@ -132,10 +132,11 @@ When the wizard asks you about the **"Old archvive dir"**, make sure to set it t
 Running ``indico-migrate``
 ++++++++++++++++++++++++++
 
-First of all, make sure that you are using the **user** and **virtualenv** created using the step **"Install Indico"**:
+First of all, make sure that you are using the **user** and **virtualenv** created using the step **"Install Indico"** and that the legacy dir is owned by this **user**:
 
 .. code-block:: shell
 
+    chown -R indico /opt/indico-legacy
```

Comment:
And `chown -R indico /opt/indico-legacy/db` otherwise I get
```
IOError: [Errno 13] Permission denied: '/opt/indico-legacy/db/Data.fs.lock'
IOError: [Errno 13] Permission denied: '/opt/indico-legacy/db/Data.fs.tmp'
IOError: [Errno 13] Permission denied: '/opt/indico-legacy/db/Data.fs'
...
```